deploy-manage/production-guidance/kibana-in-production-environments.md

```diff
@@ -1,86 +1,126 @@
-
-While {{kib}} isn’t terribly resource intensive, we still recommend running {{kib}} separate from  your {{es}} data or master nodes. To distribute {{kib}} traffic across the nodes in your {{es}} cluster, you can configure {{kib}} to use a list of {{es}} hosts.
-
-::::{warning} 
-{{kib}} does not support rolling upgrades, and deploying mixed versions of {{kib}} can result in data loss or upgrade failures. Please shut down all instances of {{kib}} before performing an upgrade, and ensure all running {{kib}} instances have matching versions.
+Additionally, the task manager enables distributed coordination across multiple Kibana instances, allowing Kibana to function as a logical cluster in certain aspects.
 
+::::{important} 
+* {{kib}} does not support rolling [upgrades](/deploy-manage/upgrade/deployment-or-cluster/kibana.md), and deploying mixed versions of {{kib}} can result in data loss or upgrade failures. Please shut down all instances of {{kib}} before performing an upgrade, and ensure all running {{kib}} instances have matching versions.
+* While {{kib}} isn’t terribly resource intensive, we still recommend running {{kib}} separate from  your {{es}} data or master nodes.
 ::::
 
+## Section overview
 
+This section provides guidance on key configurations and optimizations for running {{kib}} in production environments. You’ll learn how to scale, secure, and optimize {{kib}} for high availability and performance, as well as how to manage background tasks and other features effectively.
 
-## Load balancing across multiple {{kib}} instances [load-balancing-kibana] 
-
-To serve multiple {{kib}} installations behind a load balancer, you must change the configuration. See [Configuring {{kib}}](../deploy/self-managed/configure-kibana.md) for details on each setting.
-
-These settings must be unique across each {{kib}} instance:
+::::{note}
+Note for deployment types, as not all the sub-sections are applicable to all deployment types.
+::::
 
-```js
-server.uuid // if not provided, this is autogenerated
-server.name
-path.data
-pid.file
-server.port
-```
+Topics covered in this section:
 
-When using a file appender, the target file must also be unique:
+* [Load balancing across multiple {{kib}} instances within the same deployment](#load-balancing-kibana)
+* [Accessing multiple load-balanced {{kib}} deployments](#accessing-load-balanced-kibana)
+* [Distributing {{kib}} traffic across multiple {{es}} instances](#high-availability)
+* [Setting a memory limit to {{kib}}](#memory)
+* [Disabling legacy encryption algorithms for OpenSSL](#openssl-legacy-provider)
+* [Kibana task manager: Scaling and performance](./kibana-task-manager-scaling-considerations.md)
+* [Kibana alerting: Scaling and production considerations](./kibana-alerting-production-considerations.md)
+* [Kibana reporting production considerations](./kibana-reporting-production-considerations.md)
```

Comment:
later on, we might want to restructure this to break out these child sections into their own docs. having lots of content directly on this page, but also child pages, is a little bit of an anti-pattern

Reply:
totally agreed, it felt weird while putting all together

Reply:
On it... I think I've got it!

Reply:
done!